packages/strapi-parts/src/Badge/Badge.stories.mdx

```diff
@@ -15,7 +16,20 @@ This is the doc of the `Badge` component
 
 ## Badge
 
-Description...
+Badges are used to give a quick visual indication to the users.
+
+**Best practices**
+
+- Badge component should be filled only with numbers or a single word.
+- Only use two words if you need to describe something complex.
```

Comment:
This is in contradiction with the first point, I think

Reply:
😂 Good catch